src/Installers/Windows/AspNetCoreModule-Setup/ANCMIISExpressV2/ancm_iis_expressv2.wxs

```diff
@@ -164,22 +164,63 @@
       <DirectoryRef Id="INSTALLDIR">
         <Directory Id="INSTALLLOCATION" ShortName="ANCM" Name="$(var.ProductName)">
           <Directory Id="VersionDir" Name="$(var.ProductVersionString)">
-            <?if $(var.Platform) = "arm64" ?>
-            <Component Id="AspNetCoreModuleV2.forwarder"
-                Guid="08968573-05c1-4bf1-8879-7b818ac9525b"
-                Win64="$(var.IsWin64)">
-                <File Id="AspNetCoreModuleV2Dll.forwarder"
+            <Component Id="AspNetCoreModule" Guid="84ed6ce6-c8a3-4fa8-a872-c98a1d15dd4f" Win64="$(var.IsWin64)">
+                <?if $(var.Platform) = "arm64" ?>
+                <File Id="AspNetCoreModuleDll"
```

Comment:
You should consider setting the `File@KeyPath="yes"`. If child elements of a component do not have a keypath set, and the parent directory of the component is not a key path, then the first child is selected. In this case, it would be the file. However, if you modify the order, the key path will change. Setting it explicitly will avoid issues in the future.

This might be a good read: https://learn.microsoft.com/en-us/windows/win32/msi/what-happens-if-the-component-rules-are-broken